Makefile

```diff
@@ -105,7 +105,7 @@ GO_APIDIFF_VER := v0.7.0
 GO_APIDIFF_BIN := go-apidiff
 GO_APIDIFF := $(TOOLS_BIN_DIR)/$(GO_APIDIFF_BIN)
 
-GINKGO_VER := v2.14.0
+GINKGO_VER := $(shell go list -m -f '{{.Version}}' github.com/onsi/ginkgo/v2)
```

Comment:
I missed that the gomega bump that already merged also included this ginkgo bump, so that's why this doesn't have any go.mod changes anymore. This change is still required to keep the Makefile in sync though.

Reply:
#4491

Reply:
hmm, seems like we should probably skip the ginkgo check if the gomega does it already

Reply:
That was likely a one-off thing for this particular update. I wouldn't expect that to happen every time.

Reply:
ACK